core/server/common/src/main/java/alluxio/master/journal/raft/BufferedJournalApplier.java

```diff
@@ -204,9 +204,9 @@ public CatchupFuture catchup(long sequence) {
       Preconditions.checkState(!mResumeInProgress, "Resume in progress");
       Preconditions.checkState(mCatchupThread == null || !mCatchupThread.isAlive(),
           "Catch-up task in progress.");
-      Preconditions.checkState(sequence >= 0, String.format("Invalid sequence: %d", sequence));
-      Preconditions.checkState(mLastAppliedSequence <= sequence, String.format(
-          "Can't catchup to past. Current: %d, Requested: %d", mLastAppliedSequence, sequence));
+      Preconditions.checkState(sequence >= 0, "Invalid sequence: %d", sequence);
```

Comment:
Somewhere, can we add that it is invalid because it is less than 0? There are many ways for something to be invalid.